### PR TITLE
AC Automated Test - LRAC-8505 | master

### PR DESCRIPTION
**Pull request by manuferreira**
https://issues.liferay.com/browse/LRAC-8505

### CODE REVIEW

**Comment by manuferreira** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsOverviewAttributeBreakdown.testcase [hunk 1]
```diff
@@ -38,12 +43,80 @@ definition {
...
+				userEmailAddress = "ac1@liferay.com",
+				userFirstName = "ac1",
+				userLastName = "ac1",
+				userScreenName = "ac1");
+
+			JSONUser.addUser(
+				userEmailAddress = "ac2@liferay.com",
+				userFirstName = "ac2",
+				userLastName = "ac2",
+				userScreenName = "ac2");
```
[the remark this answers is not in the document]
Comment (alexcandidos) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsOverviewAttributeBreakdown.testcase:64]: According to @ViniLopes20 , this is the default date when no birth dates are set.

**Comment by alexcandidos** [comment]
Hi @manuferreira , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**